Dockerfile

```
@@ -95,7 +95,7 @@ RUN mkdir /static ; STATIC_DIR='/static' ./tools/sync-web-ui.sh
 FROM debian:12-slim AS qdrant
 
 RUN apt-get update \
-    && apt-get install -y ca-certificates tzdata \
+    && apt-get install -y ca-certificates tzdata libunwind8 \
```

I suggest we use `libunwind-dev` here as well.

```suggestion
    && apt-get install -y ca-certificates tzdata libunwind-dev \
```

---

it is too big

---

This stage is just used as runtime of course. Thanks!